overlays/docker-compose.monitoring.yml

```diff
@@ -0,0 +1,114 @@
+x-loki-logging: &loki-logging
+  driver: loki
+  options:
+    loki-url: "http://host.docker.internal:3100/loki/api/v1/push"
+
+x-healthcheck-options: &healthcheck-options
+  interval: 10s
+  timeout: 3s
+  retries: 3
+  start_period: 30s
+
+services:
+  app:
+    environment:
+      MONITORING_API_ENABLED: on
+      MONITORING_JVM_ENABLED: on
+      MONITORING_DBPOOL_ENABLED: on
+      MONITORING_HIBERNATE_ENABLED: on
+      MONITORING_UPTIME_ENABLED: on
+      MONITORING_CPU_ENABLED: on
+    logging: *loki-logging
+
+  database:
+    logging: *loki-logging
+
+  traefik:
+    logging: *loki-logging
+
+  # This is needed to avoid permission issues with the loki container when running as non-root user
+  loki-init:
+    image: busybox:1.37.0
+    user: root
+    volumes:
+      - loki:/loki
+    command: ["sh", "-c", "mkdir -p /loki/chunks /loki/rules /loki/wal /loki/boltdb-shipper-compactor && chown -R nobody:nobody /loki"]
+
+  loki:
+    image: grafana/loki:${LOKI_VERSION:-2.9.0}
+    restart: unless-stopped
+    user: nobody:nobody
+    security_opt:
+      - no-new-privileges:true
+    cap_drop:
+      - ALL
+    volumes:
+      - loki:/loki
+    ports:
```

Comment:
You and I both know why this port needs to be open but others might not. Please add a comment here about the driver not running on the same network as the services